src/dataflow/core/lispress.py

```diff
@@ -46,6 +46,22 @@
 Lispress = Sexp
 
 
+def try_round_trip(lispress_str: str) -> str:
```

Comment:
moved this from the `dataflow.leaderboard.evaluate` so it can be used in `dataflow.onmt_helpers` as well.